reflex-dom-core/src/Reflex/Dom/Widget/Input.hs

```diff
@@ -437,9 +437,12 @@ dropdown k0 options (DropdownConfig setK attrs) = do
         let xs = fmap (\(ix, (k, v)) -> ((ix, k), ((ix, k), v))) $ zip [0::Int ..] $ Map.toList os
         in (Map.fromList $ map snd xs, Bimap.fromList $ map fst xs)
   modifyAttrs <- dynamicAttributesToModifyAttributes attrs
+  postBuild <- getPostBuild
+  let setSelection = attachPromptlyDynWithMaybe (flip Bimap.lookupR) ixKeys $
```

Comment:
Per what @3noch said in https://github.com/reflex-frp/reflex-dom/pull/350#discussion_r362642186, we should think more about whether this is actually needed. We should do that in a separate change, however.